lib/web_ui/lib/src/engine.dart

```diff
@@ -203,6 +203,7 @@ void initializeEngine() {
   // This extension does not need to clean-up Dart statics. Those are cleaned
   // up by the compiler.
   developer.registerExtension('ext.flutter.disassemble', (_, __) {
+    window.resetHistory();
```

Comment:
Hi @mdebbar is there a way to test this? I couldn't find an example to trigger hard reload during the test.

Reply:
I don't think there's a way to test hot restart yet. Maybe @yjbanov has some thoughts?